pallets/pips/src/benchmarking.rs

```diff
@@ -174,12 +174,9 @@ fn propose_verify<T: Config>(url: Url, description: PipDescription) -> DispatchR
 
 fn execute_verify<T: Config>(state: ProposalState, err: &'static str) -> DispatchResult {
     if Proposals::<T>::contains_key(PipId(0)) {
-        assert_eq!(
-            state,
-            Module::<T>::proposals(PipId(0)).unwrap().state,
-            "{}",
-            err
-        );
+        Module::<T>::proposal_state(PipId(0)).map(|pip_state| {
+            assert_eq!(state, pip_state, "{}", err);
+        });
```

Comment:
Or do this:
```suggestion
        assert_eq!(
            Some(state),
            Module::<T>::proposal_state(PipId(0)),
            "{}",
            err
        );
```
Since we want this check to fail for `None` and if the pip's state is wrong.